share/normalize/oai.py

```diff
@@ -120,7 +120,15 @@ class OAICreativeWork(Parser):
 
     rights = tools.Join(tools.Maybe(tools.Maybe(ctx.record, 'metadata')['oai_dc:dc'], 'dc:rights'))
 
-    language = tools.ParseLanguage(tools.Maybe(tools.Maybe(ctx.record, 'metadata')['oai_dc:dc'], 'dc:language'))
```

Comment:
Could you use the `TryLink` add in [this PR](https://github.com/CenterForOpenScience/SHARE/pull/283)
You'll be able to do something like `tools.Try(ctx.record[metadata][oai_dc:dc][dc:language][0])`